Tests/XCUITests/TabsPerformanceTests.swift

```diff
@@ -5,16 +5,15 @@
 import XCTest
 
 class TabsPerformanceTest: BaseTestCase {
-    let fixtures = ["testPerfTabs_1_20startup": "tabsState20.archive", "testPerfTabs_3_20tabTray": "tabsState20.archive", "testPerfTabs_2_1280startup": "tabsState1280.archive", "testPerfTabs_4_1280tabTray": "tabsState1280.archive"]
+    let fixtures = ["testPerfTabs_1_20startup": "tabsState20.archive", "testPerfTabs_3_20tabTray": "tabsState20.archive", "testPerfTabs_2_1280startup": "tabsState1280.archive", "testPerfTabs_4_1280tabTray": "tabsState1280.archive", "testPerfHistory100startUp": "testHistoryDatabase100-places.db", "testPerfHistory100openMenu": "testHistoryDatabase100-places.db", "testPerfBookmarks1000openMenu": "testBookmarksDatabase1000-places.db", "testPerfBookmarks1000startUp": "testBookmarksDatabase1000-places.db"]
 
     override func setUp() {
         // Test name looks like: "[Class testFunc]", parse out function name
         let parts = name.replacingOccurrences(of: "]", with: "").split(separator: " ")
         let functionName = String(parts[1])
         let archiveName = fixtures[functionName]
-
         // defaults
-        launchArguments = [LaunchArguments.PerformanceTest, LaunchArguments.SkipIntro, LaunchArguments.SkipWhatsNew, LaunchArguments.SkipETPCoverSheet, LaunchArguments.SkipContextualHints]
+        launchArguments = [LaunchArguments.PerformanceTest, LaunchArguments.SkipIntro, LaunchArguments.SkipWhatsNew, LaunchArguments.SkipETPCoverSheet, LaunchArguments.SkipContextualHints, LaunchArguments.LoadDatabasePrefix + fixtures[functionName]!]
```

Comment:
Adding the database here does not work, if you look at the logs the testAppStarted crashes. That is because is not in the fixtures list. That's why we have the if condition below in L19.

Reply:
ah, i see. i thought i was seeing issues running it locally and the changes fixed it, but it must have been dumb luck.